mautic/mautic
# Email attachment tooltip with info about tracking added

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.0 for bug fixes 
| Bug fix?                               | yes
| New feature?                           | Maybe an enhancement?
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | In the tooltip
| Related developer documentation PR URL | /
| Issue(s) addressed                     | Fixes #6499




#### Description:

Just adding a new tooltip to the email attachment field requested by a marketer.

<img width="472" alt="Screen Shot 2020-10-28 at 20 15 51" src="https://user-images.githubusercontent.com/1235442/97485428-69571d80-195a-11eb-9bba-f6b4d5b48ca4.png">




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go to any email edit (or new) page, Advanced tab and check the tooltip is there.
